Topic_3/shoppingCart/src/main/java/com/mycompany/app/shoppingCart/DataBaseStub.java

```diff
@@ -0,0 +1,36 @@
+package com.mycompany.app.shoppingCart;
+
+import java.util.ArrayList;
+
+public class DataBaseStub {

```

Comment:
Minor for this topic: This is not a right way to do this because you created a catalog and should be a singleton.